lib/AST/GenericSignature.cpp

```diff
@@ -456,8 +457,7 @@ GenericSignatureImpl::getConformsTo(Type type) {
 }
 
 bool GenericSignatureImpl::conformsToProtocol(Type type, ProtocolDecl *proto) {
```

Comment:
This method is crying to be renamed to `requiresProtocol` despite the FIXIT it contains.